gormapplication/application.go

```diff
@@ -51,7 +51,7 @@ const (
 
 func NewGormDB(db *gorm.DB) *GormDB {
 	val := new(GormDB)
-	val.db = db
+	val.db = db.Set("gorm:save_associations", false)
```

Comment:
nice :) So, we were inadvertently updating records all this while.

Reply:
Yeah... caught this with Shane when adding tests for this PR. It's not only caused a lot of unnecessary updated on related tables but also caused issues with wrong values in the tables!